WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -104,20 +107,10 @@ def index
   end
 
   def create
-    new_competition_params = params.require(:competition).permit(:name, :competition_id_to_clone)
-    @competition = Competition.new(new_competition_params.merge(
-                                   registration_open: 1.week.from_now,
-                                   registration_close: 2.weeks.from_now))
```

Comment:
Oh, actually it doesn't seem to be necessary. I just assumed there was a reason for that. Nevermind!